PY-SDK/public/src/pyaver/user_market.py

```diff
@@ -192,7 +194,10 @@ async def load_multiple_by_uma(
         """
         #Just use the first program to load. This should not matter for the purpose of parsing IDLs
         #TODO - review
-        res: list[UserMarketState] = await aver_client.programs[0].account['UserMarket'].fetch_multiple(pubkeys)
+        # res: list[UserMarketState] = await aver_client.programs[0].account['UserMarket'].fetch_multiple(pubkeys)
+        account_buffers = await load_multiple_bytes_data(aver_client.connection, pubkeys, [], False)
+        res: list[UserMarketState] = UserMarket.parse_multiple_user_market_state(account_buffers, aver_client)
+        # TODO parse with version
```

Comment:
I believe the parse_multiple_user_market_state already parses with version. So we can remove the comment